router-tests/testdata/queries/products.json

```diff
@@ -22,7 +22,8 @@
         "clientLanguages": [
           "RUST",
           "TYPESCRIPT"
-        ]
+        ],
+        "unicode": "😀♥!\u0015\u0010🝥"
```

Comment:
This won't reproduce the error. You can use `"\u0015"`

Reply:
you should have tried
```
Unicode: "\U0001F600\u2665\u0021\u0015\u0010\U0001F765"
```

Reply:
👍

Reply:
I didn't understand the test in the first place. You extended the test query and updated the fixture. Now, I understand. Thanks.